util/rbac/rbac.go

```diff
@@ -343,6 +372,15 @@ func (e *Enforcer) newInformer() cache.SharedIndexInformer {
 	return v1.NewFilteredConfigMapInformer(e.clientset, e.namespace, defaultRBACSyncPeriod, indexers, tweakConfigMap)
 }
 
+// newInformers returns an informer which watches updates on the rbac configmap
+func (e *Enforcer) newAdditionalInformer() cache.SharedIndexInformer {
+	tweakConfigMap := func(options *metav1.ListOptions) {
+		options.LabelSelector = "argocd.argoproj.io/cm-type=additional-rbac"
```

Comment:
What happens if I remove the label? Will the policy be dropped from the combined policy?

Reply:
since we are using `v1.NewFilteredConfigMapInformer` with our specified labelselector, it will drop the policy from combined policy:

`time="2022-11-21T23:04:23Z" level=info msg="RBAC Additional ConfigMap 'argocd-rbac-cm-extra' deleted" security=2
`